.github/workflows/release.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
         with:
           fetch-depth: 0
-      - run: git fetch --tags --force origin # WA: https://github.com/actions/checkout/issues/882
+      - run: git fetch --tags --force origin  # WA: https://github.com/actions/checkout/issues/882
```

Comment:
NIT: Is this extra whitespace intentional?

Reply:
Yes, it's coming from yamllint, in the 0.24.7 version of tardigrade-ci, and its default rules for comments, https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.comments. In particular, this one:

```
    min-spaces-from-content: 2
```